pkgs/halo2-eth-membership-worker/src/utils.ts

```diff
@@ -27,3 +27,14 @@ export function hexToLittleEndianBytes(hex, size) {
   // Reverse the byte order for little-endian
   return new Uint8Array(bytes).reverse()
 }
+
+// @src https://github.com/RiverRuby/halo2-lib-wasm/blob/9e22a0452add207800a4aa585e345cc9fb50a549/browser/lib/halo2Prover/halo2Prover.ts#L3C1-L11C3
+export const fetchKzgParams = async (k: number) => {
+  const response = await fetch(
```

Comment:
based on my try to fetch files between 11 and 19, these files can get quite big.
```
k=11
257KiB
k=12
513KiB
k=13
1.1MiB
k=14
2.1MiB
k=15
4.1MiB
k=16
8.1MiB
k=17
17MiB
k=18
33MiB
k=19
65MiB
```

so we should cache them.
we can do this later

Reply:
Yeah I agree we can do this later; for 15 though is not too big 4.1MiB, so that should be fine.

Reply:
Yes i agree 4 MiB in not so big.

The easiest is to commit that file